.circleci/config.yml

```diff
@@ -70,11 +70,19 @@ commands:
           command: |
             echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
             LILY_IMAGE_TAG=${CIRCLE_TAG//\//-} make docker-mainnet-push
+            # omit release candidates from pushing latest
+            if [[ ! "$CIRCLE_TAG" =~ -rc[0-9]+$ ]]; then
+              LILY_IMAGE_TAG=latest make docker-mainnet-push
```

Comment:
Sure. CIRCLE_TAG is provided by circleCI and is set to the current tag which was just pushed.

We set `LILY_IMAGE_TAG` to `latest` (this is used in `push-docker-tags.sh`).

`docker-mainnet-push` calls `docker-mainnet` which sets:
```
docker-mainnet: LILY_DOCKER_FILE ?= Dockerfile
docker-mainnet: LILY_NETWORK_TARGET ?= mainnet
docker-mainnet: LILY_IMAGE_TAG ?= $(COMMIT)
```

LILY_IMAGE_TAG is not set to `$(COMMIT)` above because `?=` means "only assign if empty". Since it was previously set on entrance to `make` this line is effectively ignored.

and then calls `docker-tag-and-push-template` which calls `./scripts/push-docker-tags.sh $(LILY_IMAGE_NAME) $(LILY_IMAGE_TAG)`. `LILY_IMAGE_NAME` is set at the top of Makefile to `filecoin/lily` and `LILY_IMAGE_TAG` is set to `latest`.